src/dbos-runtime/cli.ts

```diff
@@ -137,6 +138,13 @@ program
     await runAndLog(migrate);
   });
 
+program
```

Comment:
I'd rather have "dbos postgres start" and "dbos postgres stop" commands than just "dbos postgres". That's roughly the interface users will expect imo.

Reply:
👍